AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/CampaignAdminController.cs

```diff
@@ -39,7 +39,7 @@ private ViewResult WithTenants(ViewResult view)
         // GET: Campaign/Create
         public async Task<IActionResult> Create()
         {
-            return await Task.Run(() => WithTenants(View()));
+            return await Task.Run(() => WithTenants(View("Edit")));
```

Comment:
A better way to do this is to use Task.FromResult:

Task.FromResult(WithTenants(View("Edit"));

That way, the compiler doesn't generate all the async state machine, and this simply always returns a completed task.